packages/datagateway-dataview/public/datagateway-dataview-settings.example.json

```diff
@@ -68,22 +59,40 @@
       "order": 0
     },
     {
-      "section": "Data",
+      "section": "Browse",
       "link": "/browse/investigation",
       "displayName": "DataView",
       "order": 1
     },
     {
-      "section": "Data",
+      "section": "Browse",
       "link": "/browse/proposal",
       "displayName": "DLS View",
       "order": 2
     },
     {
-      "section": "Data",
+      "section": "Browse",
       "link": "/browse/instrument",
       "displayName": "ISIS View",
       "order": 3
+    },
+    {
+      "section": "Browse",
+      "link": "/my-data/DLS",
+      "displayName": "My Data DLS",
+      "order": 4
+    },
+    {
+      "section": "Browse",
+      "link": "/my-data/ISIS",
+      "displayName": "MY Data ISIS",
+      "order": 5
+    },
+    {
+      "section": "Browse",
+      "link": "/browseStudyHierarchy/instrument",
+      "displayName": "Studies",
+      "order": 6
```

Comment:
I know these are just an example, but I think 'Studies' should be moved in between the 'view' and 'my data' headings as this is how they appear on the mock-up.